<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-P3VSL9N" failures="1" tests="1" name="Default test(failed)" time="0.064" errors="0" timestamp="2024-01-16T12:57:49 IST">
  <testcase classname="testNG.Testdemo1" name="HardAssertion" time="0.015">
    <failure type="java.lang.AssertionError" message="Fil expected [Wlcome] but found [Welcome]">
      <![CDATA[java.lang.AssertionError: Fil expected [Wlcome] but found [Welcome]
at testNG.Testdemo1.HardAssertion(Testdemo1.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- HardAssertion -->
  <testcase classname="testNG.Testdemo1" name="searchproducttest" time="0.0">
    <ignored/>
  </testcase> <!-- searchproducttest -->
  <testcase classname="testNG.Testdemo1" name="registertest" time="0.0">
    <ignored/>
  </testcase> <!-- registertest -->
  <testcase classname="testNG.Testdemo1" name="addtocartrtest" time="0.0">
    <ignored/>
  </testcase> <!-- addtocartrtest -->
</testsuite> <!-- Default test(failed) -->
